python/tvm/autotvm/utils.py

```diff
@@ -111,7 +112,7 @@ def pool_map(func, args, batch_size, verbose=False, pool=None):
 
     ret = None
     tic = time.time()
-    local_pool = pool or multiprocessing.Pool()
+    local_pool = pool or PopenPoolExecutor()
```

Comment:
same as above. No specific number of processes here. We may consider adding a default number of process getting created in the implementation of `PopenPoolExecutor`

Reply:
Same as above.